chore: DIP4 Adjusted Coinbase description to v20 (#148)
dashpay/dips: dip-0004.md

| ----- | ---- | ---- | ----------- |
| merkleRootQuorums | uint256 | 32 | Merkle root of currently active LLMQs |

Starting with version >= 3, the following fields are added:

| Field | Type | Size | Description |
| ----- | ---- | ---- | ----------- |
| bestCLHeightDiff | varint | 4 | Number of blocks between the current block and the block locked with `bestCLSignature`. |
| bestCLSignature | BLSSig | 96 | Newest ChainLock signature known by the miner. |

> | creditPoolBalance | int64_t | 8 | Total amount of locked credits at the block height. |

Reviewer: Confirmed on slack that this is DASH denominated

> | creditPoolBalance | int64_t | 8 | Total amount Dash Platform credits locked at the block height. |

Reviewer: This balance is denominated in DASH so maybe smth like this?
```suggestion
| creditPoolBalance | int64_t | 8 | Total amount DASH currently locked as of this block height. |
```

More information on the fields `bestCLHeightDiff` and `bestCLSignature` can be found in [DIP0029 - Randomness Beacon For LLMQ Selection](https://github.com/dashpay/dips/blob/master/dip-0029.md#change-to-the-coinbase-transaction).

> More information on the field `creditPoolBalance` can be found in [DIP0027 - Platform Credit Funding and Withdrawals](https://github.com/dashpay/dips/blob/master/dip-0027.md).

Reviewer: Let's leave out a link for now and add it once it's available.
```suggestion
More information on the `creditPoolBalance` field will be described in a future document.
```
